projects/storefrontlib/src/shared/components/popover/popover.component.html

```diff
@@ -1,15 +1,17 @@
 <div class="arrow"></div>
 <div class="popover-body" [cxFocus]="focusConfig">
-  <button
-    *ngIf="displayCloseButton"
-    type="button"
-    class="close"
-    (keydown.enter)="close($event)"
-    (keydown.space)="close($event)"
-    (click)="close($event)"
-  >
-    <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
-  </button>
+  <div class="close-row">
```

Comment:
the class name should follow the cx-name structure. So it should be "cx-close-row"